wordpress-mobile/WordPress-Android
# Fix #10747: Gutenberg rollout phase 2

Fixes #10747 

<s>This FluxC PR must be merged first: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1466</s> ✅ FluxC version was bumped to 1.5.6 in 7c586ea 

This PR adds:
- A new rollout phase. This target users who have aztec enabled before the first rollout phase.
- A new popup that shows only the first time the user open Gutenberg after the site has been switched from aztec to gutenberg. Popup text is: `We made big improvements to the block editor and think it\'s worth a try! We enabled it for new posts and pages but if you'd like to change to the classic editor, go to 'My Site' > 'Site Settings'`

Note: there is one downside to this, if the user wasn't part of the first rollout (it means they switched manually to Gutenberg a while ago) and they switch a site to Aztec, the next time they re-open the app this site will be switched back to Gutenberg (this will happen only once, they'll be considered in the rollout group after that).

To test:
- Set `GB_ROLLOUT_PERCENTAGE_PHASE_2` to 100% and build.
- Clear App DB, run the app, go to site settings switch a site to Aztec (this should ensure that you're not part of the first rollout).
- Close the app / re-open the app. 
- Create a new post, Gutenberg should open with the new popup.
